Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -191,8 +191,9 @@ public class MobilityCommonBridge extends HyperBridge {
     // Others
     private LottieAnimationView animationView;
     protected Method[] methods = null;
-
     protected  Receivers receivers;
+    protected static int animationDuration = 400;
+    protected static JSONObject locateOnMapConfig = null;
```

Comment:
don't use static variables. if is required to be static then make sure it is removed in reset() call.

Reply:
static is required, will add in reset